package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@capsulajs/capsulahub-core",
```

Comment:
I don't know if name useful here because it's lerna, in examples usually I see in root package.json only scripts related to lerna and `private: true` all other stuff like a name, description etc..will be different in each of nested packages.

Example
https://github.com/Quramy/lerna-yarn-workspaces-example/blob/master/package.json